### PR TITLE
Android: Update to 2.6.0, addEventListener

### PR DESCRIPTION
**Pull request by m1ga**
* update to Lottie 2.6.0-rc3
* adding `addEventListener()` back again (removing the update and complete callback)
* update event will pass the `frame` instead of `time` and `percentage` since its better to work with 
* clang all files
* fixed complete-callback being called twice
* added missing autoStart/start property e.g. so you can start it later in a listview when setting the items

Test binary: [Beta version](https://github.com/m1ga/ti.animation/releases/download/untagged-d0391cc19ebd1867fc8f/ti.animation-android-2.1.0.zip)

### CODE REVIEW

**Comment by hansemannn** [comment]
@m1ga Is there iOS parity required here?

**Comment by m1ga** [comment]
@hansemannn you already have the `complete` eventListener. In Android it got removed when I refactored the code some time ago :blush: 

We have to sync the callback parameters once I have everything ready.

The `update` event would be good event to have on iOS too. Helps to stop it at a certain frame

File: android/src/ti/animation/LottieViewProxy.java [hunk 1]
```diff
@@ -137,26 +149,78 @@ public void stop() {
...
 		}
 	}
 
-	@Kroll.setProperty @Kroll.method
-	public void setText(String layer, String text) {
+	// clang-format off
+	@Kroll.setProperty
+	@Kroll.method
+	public void setText(String layer, String text)
+	// clang-format on
```
Comment (hansemannn) [inline android/src/ti/animation/LottieViewProxy.java]: Thanks for adopting the format!

**Comment by m1ga** [comment]
released this since I'm using it in an app now and can test it better :smile: 
And it will raise the Lottie version. Now we can use this to add new features like the zip file support!

**Pull request merged**